src/utils/parse-comment/index.js

```diff
@@ -81,12 +81,13 @@ function parseAddComment(doc, action) {
 
     const who = whoMatched.startsWith('@') ? whoMatched.substr(1) : whoMatched
 
-    let contributions = doc
+    const contributions = doc
         .match('#Contribution')
         .data()
         .map(data => {
             // This removes whitespace, commas etc
             let type = data.normal
+            if (type !== 'ideas' && /s$/.test(type)) type = type.slice(0, -1) //-> singular
```

Comment:
I think this falls down in a number of cases too (e.g. _ing_ plurals, documenting/documentation/docs). If we were writing something to account for 1000s of words I'd say we keep down this road.

As we are only dealing with around 20 base terms, I think it far more accurate to just account for the variations ourselves: https://github.com/all-contributors/all-contributors-bot/pull/53/files#diff-b5ed782af77775c7d732fcefd331dd6cR39

Reply:
Yeah true.